### DISCUSSION TITLE
wiki.js维基实验、位于discord的科幻社区实验、开源知识图谱实验、web3带激励的检索系统&未来社区实验（持续更新）

### DISCUSSION THREAD

**Discussion by askender** [qihuju/qihuju.github.io#6] [Ideas]
1. wiki.js维基实验     我个人认为，维基百科所使用的mediawiki较为繁琐， wiki.js看起来更简洁。   **如果有人感兴趣，可以体验一下编辑  http://wiki.anwensf.com/   看看wiki.js 这个工具是否好用， 好用的话可以为奇忽局再搭建一个也行**
2.  位于discord的科幻社区    名称待定    **欢迎加入  https://discord.gg/umevFmTBuH**
3.   知识图谱-脑图 的进展，我会在科幻discord里更新       当前用的UI是： http://www.visualdataweb.de/webvowl/#    目前完成markdown2webvowl 转换以及呈现的实验；wikipieda-xml2markdown实验
4. web3带激励的检索系统&未来社区   目前有协议实验室的dev grant，当前的开发会重写和重构  https://anwen.cc/  (也考虑参考各种开源的论坛项目和成功的社区系统)     最终的效果：A. 能检索ipfs/filecoin上的内容，用户能添加内容的metainfo（带激励的licence等各种许可，也可以兼容CC），也考虑使用https://iscn.io/     B. 也会发放likecoin/filecoin 空投    C. 软件开源（方便其他人搭建面向未来的社区/论坛/知识协作系统）  

4 作为目前的重心刚开始开发，其中会包含1 2 3的实验成果。  

也期待奇忽局的github社区实验顺利~

**Comment by Lupingyue** [comment]
这一组问题看似比较专业，等待其他几位高手开回应。
